torch/distributed/distributed_c10d.py

```diff
@@ -753,6 +784,61 @@ def recv(tensor,
         return src
 
 
+@contextlib.contextmanager
+def _group_manager(backend):
```

Comment:
nit: _group_manager could be confused with process group. maybe _batch_p2p_manager ?